Python/Tests/Analysis/BaseAnalysisTest.cs

```diff
@@ -83,6 +85,14 @@ public BaseAnalysisTest(IPythonInterpreterFactory factory)
             return AnalysisLimits.GetDefaultLimits();
         }
 
+        protected virtual bool SupportPython3 {

```

Comment:
This should be "SupportsPython3" per standard "must answer a question" naming rule for bool properties.